### ISSUE TITLE
Support specifying method and argument index pairs in piranha.properties

### ISSUE THREAD

**Issue by lazaroclapp** [uber/piranha#6]
This is needed, e.g. to remove methods under `emptyMethods=...` for which the experiment symbol is not the first parameter.

**Comment by jan25** [comment]
Hey @lazaroclapp!
Just to clarify, do you mean like when we have
```
if(expt.flagEnabled(SOME_OTHER_ARG, TestExperimentName.SAMPLE_STALE_FLAG)) {
        System.out.println("Hello World");
}
```
We could specify `emptyMethods=(flagEnabled,1),others..` ?

**Comment by mkruber** [comment]
`emptyMethods` do not return a boolean value. Otherwise, your understanding is correct. 

Currently, the expectation is that the first argument for all the flag APIs correspond to the flag name. This can be relaxed by providing this method, argument index pairs in piranha.properties and updating the implementation to respect this.

**Comment by lazaroclapp** [comment]
Closed by #39
